packages/genesys-spark-components/src/components/stable/gux-table-toolbar/gux-table-toolbar.tsx

```diff
@@ -138,8 +139,11 @@ export class GuxTableToolbar {
     }
   }
 
-  get allFilterContextual(): HTMLGuxTableToolbarCustomActionElement[] | null {
-    return this.filterActions?.concat(this.contextualActions);
+  get allFilterContextual(): HTMLGuxTableToolbarCustomActionElement[] {
+    return [
+      ...(this.filterActions || []),
+      ...(this.contextualActions || [])
+    ].filter(action => action != null);
   }
```

Comment:
So I should have described this more, but would making the same changes as `get permanentActions` in `get contextualActions` and `get filterActions` be better? That way it removes the null typing entirely and no changes are needed in `get allFilterContextual`.

Also could turn them into a tertiary depending on how you feel.
i.e.

```
get permanentActions(): HTMLGuxTableToolbarCustomActionElement[] {
    return this.permanentSlot?.hasChildNodes ?
        Array.from(
             this.permanentSlot?.querySelectorAll(
                 'gux-table-toolbar-action, gux-table-toolbar-custom-action'
             )
       )
       : [];
}
```

Reply:
Yeah this makes sense to me 👍